implementations/code_df/open_issue_age.py

```diff
@@ -0,0 +1,108 @@
+from datetime import datetime
+
+import utils
+from issue import Issue
+
+
+class OpenIssueAge(Issue):
+    """
+    Class for Age of Open Issues
+    """
+
+    def _flatten(self, item):
+        """
+        Flatten a raw issue fetched by Perceval into a flat dictionary.
+
+        A list with a single flat directory will be returned.
+        That dictionary will have the elements we need for computing metrics.
+        The list may be empty, if for some reason the issue should not
+        be considered.
+
+        :param item: raw item fetched by Perceval (dictionary)
+        :returns:   list of a single flat dictionary
+        """
+
+        creation_date = utils.str_to_date(item['data']['created_at'])
+        if self.since and (self.since > creation_date):
+            return []
+
+        if self.until and (self.until < creation_date):
+            return []
+
+        # all pull requests are considered to be issues by default.
+        # this check removes pull requests from the
+        # analysis.
+        if 'pull_request' in item['data']:
+            return []
+
+        flat = {
+            'repo': item['origin'],
+            'hash': item['data']['id'],
+            'category': "issue",
+            'author': item['data']['user']['login'],
+            'created_date': creation_date,
+            'current_status': item['data']['state']
+        }
+
+        if flat['current_status'] == 'open':
+            flat['open_issue_age'] = \
+               (datetime.now() - flat['created_date']).days
+
```

Comment:
The open issue age field is calculated by simply taking the days value from the timedelta object. The `if` condition makes it so that the final `df` has only `open` issues with valid values for the `open_issue_age` column. Do you agree with this?

Reply:
I think open issue is defined only for issues that were closed... However, since that metric is still not fixed in stone, proceed as you want, and we check that later.

Reply:
Ok. If the final definition deviates from this, I'll update the implementation with another patch.

Reply:
OK.